office-add-in/manifest.xml

```diff
@@ -1,29 +1,31 @@
-<!--
-  ~ SPDX-FileCopyrightText: 2025 INFO.nl
-  ~ SPDX-License-Identifier: EUPL-1.2+
--->
-
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <OfficeApp xmlns="http://schemas.microsoft.com/office/appforoffice/1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:bt="http://schemas.microsoft.com/office/officeappbasictypes/1.0" xmlns:ov="http://schemas.microsoft.com/office/taskpaneappversionoverrides" xsi:type="TaskPaneApp">
   <Id>e5bfd847-4b36-4766-9b1b-3b6f2c4c41c3</Id>
-  <Version>0.0.0.1</Version>
-  <ProviderName>INFO</ProviderName>
+  <Version>0.0.1</Version>
+  <ProviderName>INFO.nl</ProviderName>
   <DefaultLocale>nl-NL</DefaultLocale>
-  <DisplayName DefaultValue="PodiumD Office Add-In"/>
-  <Description DefaultValue="Dimpact PodiumD office integratie add-in."/>
+  <DisplayName DefaultValue="Zaakgerichtwerken Office Add-In"/>
+  <Description DefaultValue="Zaakgerichtwerken Office integratie add-in."/>
   <IconUrl DefaultValue="https://localhost:3000/assets/icon-32.png"/>
   <HighResolutionIconUrl DefaultValue="https://localhost:3000/assets/icon-64.png"/>
   <SupportUrl DefaultValue="https://www.info.nl/"/>
   <AppDomains>
-    <AppDomain>https://www.info.nl/dimpact/podiumd-office-integratie</AppDomain>
+    <AppDomain>https://www.info.nl/zaak-gericht-werken/office-integratie</AppDomain>
   </AppDomains>
+
+  <!-- Hosts Section: Specifies which Office apps the add-in supports -->
   <Hosts>
-    <Host Name="Document"/>
+    <Host Name="Document"/>  <!-- For Word -->
+    <Host Name="Workbook"/>  <!-- For Excel -->
+    <Host Name="Presentation"/>  <!-- For PowerPoint -->
```

Comment:
Initially we will only support Word. Other office apps will follow in the future.